expan/core/statistical_test.py

```diff
@@ -0,0 +1,70 @@
+from enum import Enum
+
+
+class StatisticalTest(object):
+    """ This class describes what has to be tested against what and represent a unit of statistical testing. 
+    :param kpi_name: name of the kpi
+    :type  kpi_name: str
+    :param features: list of features used for subgroups
+    :type  features: list[FeatureFilter]
+    :param variants: variant column name and their values
+    :type  variants: Variants
+    :param formula: formula of the kpi, if this is a derived kpi
+    :type  formula: str
+    """
+    def __init__(self, kpi_name, features, variants, formula=None):
+        self.kpi_name  = kpi_name
+        self.features  = features
+        self.variants  = variants
+        self.formula   = formula
```

Comment:
The design of derived kpi is one place that might need discussion.

Reply:
An explicit class for KPIs and derived one as a subclass?

Reply:
We've thought about it too. Not sure if this might become additional complexity?

Reply:
Maybe we should first think of an interface of the `KPI` functionality

For Expan, KPIs are just columns in the data frame, don't you think? Or is there any additional functionality?

Reply:
yeah. for derived kpis, it is used for creating a new column and store the referencing column.